R/incotrend.R

```diff
@@ -84,12 +84,10 @@ toolPreftrend <- function(SWS, ptab, calibdem, incocost, years, GDP_POP_MER,
   tmps[, c("logit.exponent", "tot_price") := NULL]
   ## merge placeholder
   FVtarget <- rbind(FVtarget, tmps)
-
   FVtarget[, sw := ifelse(approx == "spline", na.spline(sw, x = year), na.approx(sw, x = year)),
            by=c("region", "vehicle_type", "technology")]
-  FVtarget[sw < 0, sw := 0]
-
-  ## introduces NA for sw == 0
+  FVtarget[sw < 0, sw := 0.0001]
```

Comment:
is this safe? Where do the negative SWs appear?

Reply:
I tried to do a spline- this setting is no longer working and we should think on removing it. I will also remove this line